pkg/security/ebpf/c/rmdir.h

```diff
@@ -94,14 +108,12 @@ SYSCALL_KRETPROBE(rmdir) {
         return 0;
 
     int retval = PT_REGS_RC(ctx);
-
     if (IS_UNHANDLED_ERROR(retval)) {
-        invalidate_inode(ctx, syscall->rmdir.path_key.mount_id, syscall->rmdir.path_key.ino, 0);
         return 0;
     }
 
     int enabled = is_event_enabled(EVENT_RMDIR);
-    if (enabled) {
+    if (!syscall->discarded && enabled) {
```

Comment:
should we not use the same test for the following invalidate_inode ?

Reply:
Indeed. Fixed